edi_oca/__manifest__.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 ACSONE
+# @author: Simone Orsi <simahawk@gmail.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "EDI",
+    "summary": """
+    Define backends, exchange types, exchange records,
+    basic automation and views for handling EDI exchanges.
+    """,
+    "version": "15.0.1.0.0",
+    "website": "https://github.com/OCA/edi",
+    "development_status": "Alpha",
```

Comment:
@simahawk  maybe Beta

Reply:
it has been done here https://github.com/OCA/edi/pull/488

Reply:
pls include this change